presto-docs/src/main/sphinx/release/release-0.277.rst

```diff
@@ -0,0 +1,31 @@
+=============
+Release 0.277
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add UDF :func:`find_first` to find the first array element which matches a predicate.
+* Fix compilation error in common subexpression elimination when duplicate expressions are present.
+* Fix enforcement of query analyzer timeout during planning.
+* Add support to fail query when number of leaf plan nodes exceeds a limit. This can be controlled with ``leaf_node_limit_enabled`` and ``max_leaf_nodes_in_plan`` session properties.
+
+ORC Changes
+-----------
+* Fix bug in ORC flat map reader causing internal errors in some cases.
```

Comment:
We can remove this.  Flat map is only supported in DWRF format and no one uses DWRF format except Meta.